src/style.css

```diff
@@ -0,0 +1,25 @@
+html {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+h2 a,
+h2 a:visited {
+  color: green;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
All fixed by your recommendations! But if I stylize classes over h1 h2 tags, I get a fail report during test. So h1, h2 I keep the same.